### PR TITLE
Added pagination header parsing to responses

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

**Comment by DenverM80** [comment]
looks good

File: ds3/ds3.py [hunk 1]
```diff
@@ -6406,61 +6406,80 @@ def __check_status_codes__(self, expected_codes):
       ds3_error = XmlSerializer().to_ds3error(self.response.read(), self.response.status, self.response.reason)
       raise RequestFailed(err, ds3_error)
 
+  def parse_int_header(self, key, headers):
+    if not headers:
+      return
```
Comment (rpmoore) [inline ds3/ds3.py]: This should return None here

Comment (RachelTucker) [inline ds3/ds3.py]: Good catch

**Comment by RachelTucker** [comment]
Updated.  All tests passing.

**Pull request merged**